src/platform_impl/linux/wayland/event_loop.rs

```diff
@@ -148,7 +148,7 @@ impl<T: 'static> EventLoop<T> {
 
         let mut seat_manager = SeatManager {
             sink: sink.clone(),
-            relative_pointer_manager_proxy: None,
+            relative_pointer_manager_proxy: Arc::new(Mutex::new(None)),
```

Comment:
Maybe I'm missing something, but why is `Arc<Mutex<_>>` necessary? Wouldn't `Rc<RefCell<_>>` be sufficient? It seems to me that the `SeatManager` and `SeatData` always remain on the same thread ad the `EventLoop`.

Reply:
Of course. That makes senses. Updated the pull request to use RefCell and Rc instead.